.github/workflows/main.yml

```diff
@@ -0,0 +1,52 @@
+name: vhost
```

Comment:
For consistency, can we use rust-vmm-ci?

Reply:
sure, we switch to use rust-vmm-ci